### DISCUSSION TITLE
Add option for gameplay skin elements to ignore UI scaling

### DISCUSSION THREAD

**Discussion by Finadoggie** [ppy/osu#26244] [Ideas]
Title says it all, add some sort of option to easily allow gameplay elements such as the HP bar to ignore UI scaling. 

Currently, changing the UI scaling breaks nearly every popular skin. Telling users to resize the HP bar in the skin isn't a good option, as that means users have to resize the HP bar every time they change to a different skin or change their UI scale. In essence, people who use "legacy" skins are incentivized not to change UI scaling at all. Adding some form of checkbox to have gameplay elements ignore UI scaling is a simple fix that would greatly improve quality of life for everyone who uses a skin which wasn't made with lazer in mind.

Preferably, the option would be something like this so it could easily be toggled on and off when switching skins.
![image](https://github.com/ppy/osu/assets/75299710/10c232e7-8cd3-49c3-88e0-ab382985659f)
I'm sure you would prefer this to be an option for skin elements themselves in the skin editor, but having the option as simple and unobfuscated as possible would be best for quick switching, which is ultimately what an option like this should be optimized for.

**Comment by GhostylGit** [comment]
This has been bothering me as well. There needs to be a separate scaling option for ui and gameplay, maybe not just a toggle for 1x but two sliders, or if that isn't preferable then an option to scale elements in the skin editor to there 1x size on right click would also solve this but it would be tedious to update every legacy skin that way.
